<?xml version="1.0" encoding="utf-8"?>
<!-- Launch file for launching spot micro motion command node on raspberry pi in combination -->
<!-- with ros-i2cpwmboard node. -->

<launch>

    <!-- Launch required ros-i2cpwmboard node. Can't just include it's launch file because it has a -->
    <!-- prefix command with sudo which screws it up in starting. Instead, manually start the node-->
    <node name="i2cpwm_board_node" pkg="i2cpwm_board" type="i2cpwm_board" output="screen">                                           â”‚walk: Start walk mode and keyboard motion control
    </node> 

    <!-- Defining the node and executable and publishing the output on terminal-->
    <node name="spot_micro_motion_cmd_node" pkg="spot_micro_motion_cmd" type="spot_micro_motion_cmd_node" output="screen">
        <!-- loading the parameters from yaml file during th runtime -->
        <rosparam command="load" file="$(find spot_micro_motion_cmd)/config/spot_micro_motion_cmd.yaml"/>
    </node>
</launch>
